plumgrid_playbooks/enable_pg_extensions.yml

```diff
@@ -39,12 +32,15 @@
         file: /tmp/GPG-KEY
         state: present
 
-    - name: Running apt-update
-      apt:
-        update_cache: yes
```

Comment:
no reason to use it as [apt_repository](https://github.com/ansible/ansible-modules-core/blob/7f55c9fc8eb05c525adeea69043620a7b3023ac4/packaging/os/apt_repository.py#L467) module has default `update_cache`